apps/web-giddh/src/app/app.module.ts

```
@@ -39,39 +39,46 @@ import { MatButtonModule } from '@angular/material/button';
-// Application wide providers
+// Get white label configuration from localStorage
+let whiteLabelConfig = JSON.parse(localStorage.getItem('whiteLabel'));
+console.log(whiteLabelConfig)
 const APP_PROVIDERS = [
     ...APP_RESOLVER_PROVIDERS,
-    { provide: APP_BASE_HREF, useValue: IS_ELECTRON_WA ? './' : AppUrl + APP_FOLDER }
+    {
+        provide: APP_BASE_HREF,
+        useValue: IS_ELECTRON_WA
+            ? './'
+            : whiteLabelConfig && whiteLabelConfig.body && whiteLabelConfig.body.giddhWhiteLabel.domainName
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)